app/services/preserved_object_handler.rb

```diff
@@ -285,6 +285,10 @@ def set_status_as_seen_on_disk(pres_copy, found_expected_version)
     end
 
     # TODO: do the check that'd set INVALID_CHECKSUM_STATUS
+    #  and actually, maybe we should either 1) trigger it async, or 2) break out the status
+    #  update, otherwise checksumming could get enclosed in a DB tx, and that seems like a real
+    #  bad idea, cuz that tx might be open a looooooong time.
```

Comment:
tx?

Reply:
oh ticket?

Reply:
oh, uh, yeah, colloquialism for transaction, i've tried to avoid that outside of my own notes since it's not always clear (case in point), but i slipped here.

whoops, will correct and re-push...

Reply:
fixed the comment